Makefile

```diff
@@ -46,6 +47,9 @@ ARCH := $(shell uname -m)
 
 COMMIT_HASH := $(shell ./misc/hash.sh)
 
+PYVERSION := $(shell echo ${COMMIT_HASH} | sed 's/^v//;s/-/+/;s/-/./g')
```

Comment:
Can we get a comment on what this is doing? Is there some kind of restriction on versions in the pip world?